docs/Technical Reference.md

```diff
@@ -198,6 +198,27 @@ In the template menu (CloudFormation or Service Catalog), select Google under *U
 
 ![](./imgs/google_auth.png)
 
+#### Active Directory
+
+1. Go to Azure Portal > Active Directory > App Registrations
+1. Click New Registration
+1. Name the app, select the Supported account types
+1. Set a Redirect URI from a "Single-page application" to
```

Comment:
Need to also enable "ID tokens (used for implicit and hybrid flows)" under "Authentication", after creating the app.

Reply:
Oh yeah, fixed